### PR TITLE
Create Standard-Dependencies-Issue-Template.md

### PR DESCRIPTION
**Pull request by HLCharlie**
Created a template for new Dependencies Issues hopefully covering all salient information

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (rashidsaf) [inline .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:2]:
```suggestion
name:  Dependencies Issue
```

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (rashidsaf) [inline .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:4]:
```suggestion
labels: dependencies
```

**Review by rashidsaf — CHANGES_REQUESTED** [review]

**Review by fortoulrodolfo — APPROVED** [review]

**Review by rashidsaf — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (BKorenek) [inline .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:30]: Are we sure we don't want a release notes here? most of the dependency updates will effect logic on the back end or UI

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (BKorenek) [inline .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:14]: Are we sure this information isn't redundent? it's all generated along side the PR with commits etc.

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (BKorenek) [inline .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:40]: Testing shouldn't be necessary for a dependency update

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (BKorenek) [inline .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:49]: Same as above

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (BKorenek) [inline .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:52]: shouldn't this bit be much sooner in the process? otherwise there would be a large gap in between announcement and deployment of code changes

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (BKorenek) [inline .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:24]: I feel like this should be the same across the board. Dependency is updated, and UX is not altered

**Review by BKorenek — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (HLCharlie) [reply .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:30]: But if implemented correctly, they should have no visible changes in the UI and no change to the actual business part of the logic, correct? If so, there is nothing to announce to a user. They just don't see it.

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (HLCharlie) [reply .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:14]: I would think that at the Issue level, we have the generic list of things in the updated version and in the PR we would have specifics about what we leveraged, if anything. In some cases we just verify that the existing code works with the new version, correct?

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (HLCharlie) [reply .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:40]: That's Unit and Integration tests  for that checkbox. Don't you have to run them before you go to Review? Maybe we need to be more specific in all the templates?

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (HLCharlie) [reply .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:49]: The test results here is sometimes just a note from Kyle that he ran a sanity test on this or that part of the system to verify that the build didn't break anything fundamental.

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (BKorenek) [reply .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:30]: Ideally, but when reporting bugs if there is a release note pertaining to the things that could be affected by the update. E.g. js-datepicker "Please let us know if you see any problems with the datepicker when selecting orders, or if a customer has issues picking the date of there lab visit for expediting orders" this would prevent bugs from being made, and paying out bounties.

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (BKorenek) [reply .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:14]: yeah, that is usually the case from what I've seen. Most will just come through as a PR that once approved by reviewers, if tests are passing, gets merged.

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (HLCharlie) [reply .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:31]: got it. Added releasenotes instructions

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (HLCharlie) [reply .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:14]: ok then no changes necessary. Issue has general list, PR has specifics.

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (HLCharlie) [reply .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:41]: Can we leave it as is? The testing is necessary but the implementation of new/modified tests should not be. Unless of course something fails. But with the new unit testing, we may decide to force creation of new unit tests...

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (HLCharlie) [reply .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:50]: cool, is resolved then.

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (HLCharlie) [reply .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:53]: hmm, somehow lost a whole bunch of comments... What I tried to say previously is that at the moment, I am limited as I do my thing in my pipeline. However, soon the new process will have me doing my approval BEFORE things go to Deploy which will address your stated problem. Now there is another one. I think the current process is to announce only after the Issue is CLOSED which is waaay after the rollout. I think announcement should be made after successful and verified deployment because that's when they see the stuff. If we have to de-announce it because FP rejects the Issue, then so be it. With the new process we can have announcement rolled into the Deployment process. For now, can we leave this?

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (HLCharlie) [reply .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:24]: hmm, lost another set of comments. Probably too senile to hit "Comment" which really should be "Save" since I already commented by writing in the box... Anywho, testing these generally relies on a sanity test of whatever area of the system QA deems necessary to test but usually with a lot of input from Dev as to what has likely been broken, er changed. I think there will be lots of cases where nothing special is needed but for those that do, it's worth it to have this section. Can I leave it?

File: .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md
Comment (BKorenek) [reply .github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md:53]: As far as I know everyone does it after deploy and test but 🤷

**Review by BKorenek — APPROVED** [review]

**Pull request merged**
